net-libs/tox/tox-0.2.8.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils systemd user
+
+MY_P="c-toxcore-${PV}"
+DESCRIPTION="Encrypted P2P, messaging, and audio/video calling platform"
+HOMEPAGE="https://tox.chat"
+SRC_URI="https://github.com/TokTok/c-toxcore/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0/0.2"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="+av daemon ipv6 no-log log-trace log-debug log-info log-warn +log-error static-libs test"
+
+REQUIRED_USE="^^ ( no-log log-trace log-debug log-info log-warn log-error )"
+
+RDEPEND="
+	av? ( media-libs/libvpx:=
+		media-libs/opus )
+	daemon? ( dev-libs/libconfig )
+	>=dev-libs/libsodium-0.6.1:=[asm,urandom,-minimal]"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+S="${WORKDIR}/${MY_P}"
+
+src_prepare() {
+	cmake-utils_src_prepare
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_TOXAV=$(usex av)
+		-DMUST_BUILD_TOXAV=$(usex av)
+		-DBOOTSTRAP_DAEMON=$(usex daemon)
+		-DENABLE_STATIC=$(usex static-libs)
+		-DENABLE_SHARED=ON
+	)
+	if use no-log; then
+		mycmakeargs+=(-DMIN_LOGGER_LEVEL="")
+	fi
+	if use log-trace; then
+		mycmakeargs+=(-DMIN_LOGGER_LEVEL="TRACE")
+	fi
+	if use log-debug; then
+		mycmakeargs+=(-DMIN_LOGGER_LEVEL="DEBUG")
+	fi
+	if use log-info; then
+		mycmakeargs+=(-DMIN_LOGGER_LEVEL="INFO")
+	fi
+	if use log-warn; then
+		mycmakeargs+=(-DMIN_LOGGER_LEVEL="WARNING")
+	fi
+	if use log-error; then
+		mycmakeargs+=(-DMIN_LOGGER_LEVEL="ERROR")
+	fi
+
+	if use test; then
+		mycmakeargs+=(-DBUILD_AV_TEST=ON)
```

Comment:
Hmm, wouldn't just `-DBUILD_AV_TEST=$(usex test)` in the main option list work? Also moving ipv6 similarly there.